README.md

```diff
@@ -15,11 +15,12 @@
 
 Add your answers to this markdown file.
 
-Three reasons what asymptotic analysis may be misleading with respect to actual performance in practice are hardware, constant factors, data. When an algorithm is ran on two seperate computers, if one computer has better hardware it will run faster. This can be misleading if the two computers are comparing times. This can be the case for two equal computers as well, just with slightly different parts, there is factors like the memory and CPU that could change results. An algorithm can ignore constant factors when using asymptotic analysis. Larger constant factors can make the algorithm slower for practical input sizes. Something else that can be misleading about asymptotic analyis is if the inputted data into an algorithm is all the same. There can be different types in input of data.
+Three reasons what asymptotic analysis may be misleading with respect to actual performance in practice are hardware, constant factors, and run times. When an algorithm is ran on two seperate computers, if one computer has better hardware it will run faster. This can be misleading if the two computers are comparing times. This can be the case for two equal computers as well, just with slightly different parts, there is factors like the memory and CPU that could change results. An algorithm can ignore constant factors when using asymptotic analysis. Larger constant factors can make the algorithm slower for practical input sizes. Something else that can be misleading about asymptotic analysis is that two algorithms may have the same asymptotic complexity but have different run times. 
 
 
-Using asymptotic complexity, if a binary search tree with 1,000 elements takes 5 seconds, a binary search tree with 10,000 elements would take roughly 3 times longer. The time complexity of a binary search tree is O(h), where h is the height. The asymptotic complexity of a tree is O(logN). If you take 10,000/1,000 you get 10. If you take log base 2 of 10, you roughly get three, whioch is the ratio of the two compared. You can multiply this value by 5 seconds to get an exact time. 
+Using asymptotic complexity, if a binary search tree with 1,000 elements takes 5 seconds, a binary search tree with 10,000 elements would take roughly 3 times longer. The time complexity of a binary search tree is O(h), where h is the height. The asymptotic complexity of a tree is O(logN). If you 10,000 elements and divide it by the elements for the case that took 5 second, 10,000/1,000 you get 10. If you take log base 2 of 10, you roughly get three, which is the ratio of the two compared. So the time of a list with 1,000 elements takes 5 seconds, and the time that a list of 10,000 elements would tak is, Log base 2 (10)= 3 * 5 seconds = 15 seconds. 
```

Comment:
Don't understand how you're getting the ratio of 3. The ratio of $\log_2 1000$ and $\log_2 10000$ is nowhere near 3.